extensions-contrib/time-min-max/src/main/java/io/druid/query/aggregation/TimestampAggregatorFactory.java

```diff
@@ -86,6 +87,12 @@ public Object combine(Object lhs, Object rhs)
   }
 
   @Override
+  public AggregateCombiner makeAggregateCombiner()
+  {
+    throw new UOE("[%s] is not supported during ingestion for rollup", getClass().getSimpleName());
```

Comment:
Ok, I added. Though I'm not sure there is a way currently to use it. And if anything, because of #4658 I doubt that anybody used it. (Also fixed that bug)